src/CalculatorUITestFramework/StandardCalculatorPage.cs

```diff
@@ -14,29 +19,39 @@ public class StandardCalculatorPage
         private WindowsDriver<WindowsElement> session => WinAppDriver.Instance.CalculatorSession;
         public StandardOperatorsPanel StandardOperators = new StandardOperatorsPanel();
         public MemoryPanel MemoryPanel = new MemoryPanel();
+        public CalcMemoryFlyout CalcMemoryFlyout = new CalcMemoryFlyout();
         public HistoryPanel HistoryPanel = new HistoryPanel();
+        public CalcHistoryFlyout CalcHistoryFlyout = new CalcHistoryFlyout();
         public NavigationMenu NavigationMenu = new NavigationMenu();
         public WindowsElement Header => this.session.TryFindElementByAccessibilityId("Header");
-
+        public WindowsElement Minimize => this.session.TryFindElementByAccessibilityId("Minimize");
+        public WindowsElement Maximize => this.session.TryFindElementByAccessibilityId("Maximize");
+        public WindowsElement AppName => this.session.TryFindElementByAccessibilityId("AppName");
         public WindowsElement CalculatorResult => this.session.TryFindElementByAccessibilityId("CalculatorResults");
+        public WindowsElement CalculatorExpression => this.session.TryFindElementByAccessibilityId("CalculatorExpression");
+        public WindowsElement EnterAlwaysOnTopButton => this.session.TryFindElementByAccessibilityId("NormalAlwaysOnTopButton");
+        public WindowsElement ExitAlwaysOnTopButton => this.session.TryFindElementByAccessibilityId("ExitAlwaysOnTopButton");
+        public AppiumWebElement ToolTip => WinAppDriver.Instance.CalculatorSession.FindElementByClassName("ToolTip").FindElementByClassName("TextBlock");
+        public WindowsElement Window => this.session.FindElementByClassName("Windows.UI.Core.CoreWindow");
+        public WindowsElement CalculatorAlwaysOnTopResults => this.session.TryFindElementByAccessibilityId("CalculatorAlwaysOnTopResults");
 
         public void NavigateToStandardCalculator()
         {
             // Ensure that calculator is in standard mode
             this.NavigationMenu.ChangeCalculatorMode(CalculatorMode.StandardCalculator);
             Assert.IsNotNull(CalculatorResult);
         }
-
         /// <summary>
         /// Clear the Calculator display, Memory Panel and optionally the History Panel
         /// </summary>
         public void ClearAll()
         {
+            this.StandardNonKoTMode();
```

Comment:
Please use AoT to be consistent with the product code.

Reply:
Change will appear in my next commit